manager.java

```diff
@@ -1,70 +1,51 @@
-                System.out.println(printProfit);
-            } else {
-                int numOfBackRowSeats = ((int)Math.ceil(numOfRows / 2) + 1) * numOfSeats;
-                int numOfFrontRowSeats = ((int)Math.floor(numOfRows / 2)) * numOfSeats;
-                int profit = (numOfFrontRowSeats * 10) + (numOfBackRowSeats * 8);
-                String printProfit = "$" + String.valueOf(profit);
-                System.out.println(printProfit);
-            }
-        }
-    }
 
-    static void printSeatArrangement(int numOfRows, int numOfSeats) {
+    static void printSeatArrangement(int numOfRows, int numOfSeats, char[][] seatLayout) {
         System.out.println("Cinema:");
         System.out.print("  ");
-        for(int y = 1; y <= numOfSeats; y++) {
-            System.out.print(String.valueOf(y) + " ");
+        for (int y = 0; y <= numOfSeats - 1; y++) {
+            System.out.print(y + 1 + " ");
         }
         System.out.println();
-        for(int i = 1; i <= numOfRows; i++) {
-            System.out.print(String.valueOf(i) + " ");
-            for (int x = 1; x <= numOfSeats; x++) {
-                System.out.print("S ");
+        for (int i = 0; i <= numOfRows - 1; i++) {
+            System.out.print(i + 1  + " ");
+            for (int x = 0; x <= numOfSeats - 1; x++) {
+                System.out.print(seatLayout[i][x] + " ");
             }
             System.out.println();
         }
+        System.out.println();
     }
 
-    static void reserveSeat(int numOfSeats, int numOfRows, int rowNum, int seatNum) {
-        System.out.println("Cinema:");
-        System.out.print("  ");
-        for(int y = 1; y <= numOfSeats; y++) {
-            System.out.print(String.valueOf(y) + " ");
-        }
-        System.out.println();
-        for(int i = 1; i <= numOfRows; i++) {
-            System.out.print(String.valueOf(i) + " ");
-            for (int x = 1; x <= numOfSeats; x++) {
-                if (x == seatNum && i == rowNum) {
-                    System.out.print("B ");
-                } else{
-                    System.out.print("S ");
+    static char[][] reserveSeat (int numOfSeats, int numOfRows, int totalSeats, char[][] seatLayout){
+        int rowNum = rowNum() - 1;
+        int seatNum = seatNum() - 1;
+        ticketPrice(totalSeats, numOfRows, rowNum);
+        for (int i = 0; i <= numOfRows - 1; i++) {
+            for (int x = 0; x < numOfSeats; x++) {
+                if (i == rowNum && x == seatNum) {
+                    seatLayout[rowNum][seatNum] = 'B';
                 }
             }
-            System.out.println();
+        } return seatLayout;
+    }
+
+    static char[][] initializeSeats(int numOfRows, int numOfSeats, char[][] seatLayout) {
+        for (int i = 0; i <= numOfRows - 1; i++) {
+            for (int x = 0; x < numOfSeats; x++) {
+                seatLayout[i][x] = 'S';
+            }
         }
+        return seatLayout;
```

Comment:
I would consider using something like a List<> or HashMap instead of these char[][] I think you can avoid a lot of these iterations with those Collections